pkg/commands/process/settings/rules/ruby/third_parties/segment.yml

```diff
@@ -0,0 +1,45 @@
+patterns:
+  - pattern: |
+      $<SEGMENT_ANALYTICS>.$<METHOD>($<...>$<DATA_TYPE>$<...>)
+    filters:
+      - variable: SEGMENT_ANALYTICS
+        detection: segment_init
+      - variable: METHOD
+        values:
+          - alias
+          - group
+          - identify
+          - page
+          - track
+      - variable: DATA_TYPE
+        detection: datatype
+auxiliary:
+  - id: segment_init
+    patterns:
+      - |
+        Segment::Analytics.new()
+languages:
+  - ruby
+trigger: local
+severity:
+  default: low
+  PII: critical
+  PHI: medium
+  PD: high
+skip_data_types:
+  - "Unique Identifier"
+metadata:
+  description: "Do not send sensitive data to Segment."
+  remediation_message: |
+    ## Description
+    Leaking sensitive data to third-party analytics tools is a common cause of data leaks and can lead to data breaches. This rule looks for instances of sensitive data sent to Segment.
+
+    ## Remediations
+
+    When sending data to analytics libraries, ensure all sensitive data is removed.
+
+    ## Resources
+    - [Segment docs](https://segment.com/docs/connections/sources/catalog/libraries/server/ruby/)
+  dsr_id: DSR-1
```

Comment:
Here I've kept the description and remediation quite the same as the logger copy, but in this case Segment is not a logger, it's an analytics tool. 

The DSR ID is the same, I think. 

Ping @markmichon for any thoughts or suggestions  🙏

Reply:
Thanks, lgtm 🙇